lib/course_planner_web/controllers/user_controller.ex

```diff
@@ -75,4 +77,19 @@ defmodule CoursePlannerWeb.UserController do
     |> put_flash(:info, "Users notified successfully.")
     |> redirect(to: user_path(conn, :index))
   end
+
+  def resend_email(conn, %{"id" => id}) do
+    token = ControllerHelpers.random_string 48
+    url = Helpers.password_url(conn, :edit, token)
```

Comment:
is it correct ?
you are generating a new link for the user to acess the reset password while there is already one existing and saved to him. so shouldn't you first update the user token with the newly generated one then tun send_user_email ?

Reply:
You might be right, I'll test that case.

Reply:
@ghatighorias is right, this will generate an invalid url